apps/api/prisma/schema.prisma

```diff
@@ -3,21 +3,21 @@
 
 generator client {
   provider   = "prisma-client"
-  output     = "../src/generated/prisma"
+  output     = "./src/generated/prisma"
   engineType = "client"
 }
 
 datasource db {
   provider = "postgresql"
-  url      = env("DATABASE_URL")
+  schemas  = ["api"]
```

Comment:
Since we are only using a single schema per service, putting the schema name into the connection string directly might be the easier way with less boilerplate.

https://www.prisma.io/docs/orm/prisma-schema/data-model/multi-schema